### PR TITLE
update_commitment for exposed APIs of ipa_multipoint

### PR DESCRIPTION
**Pull request by agnxsh**
function added as per the design spec of this doc https://hackmd.io/bJbN9kDeSyCrMX2x3-j93g?view
feel free to recommend me any changes that are needed,

testing isn't done yet because i've not yet got `ideal` test vectors for this function

### CODE REVIEW

**Comment by agnxsh** [comment]
Need to add the `new_commitment = commitment + (newValue-oldValue)*CRS[index]` change

File: .idea/rust-verkle-dragan.iml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+<module type="EMPTY_MODULE" version="4">
+  <component name="NewModuleRootManager">
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
+      <excludeFolder url="file://$MODULE_DIR$/target" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (dragan2234) [inline .idea/rust-verkle-dragan.iml]: can you please remove unnecessary files @agnxsh

Comment (agnxsh) [reply .idea/rust-verkle-dragan.iml]: Sure sure will do

**Review by dragan2234 — APPROVED** [review]
thanks

**Pull request merged**